src/replication/replication_schema_extract.cpp

```diff
@@ -102,6 +136,11 @@ int replication_schema_extract (const char *program_name)
   output_net_index.set_buffer_type (NET_PROXY_BUF_TYPE_EXTRACT_INDEXES_END);
   output_net_index.send_to_network ();
 
+  if (error == NO_ERROR)
```

Comment:
Add comments to explain each send and dependency between sends.

Reply:
fixed